test/collection.js

```diff
@@ -1565,7 +1565,7 @@
     collection.on('sort', function() {
       ok(true);
     });
-    collection.set([{id: 3}, {id: 2}, {id: 1}, {id: 0}]);
+    collection.set([{id: 1}, {id: 2}, {id: 3}, {id: 0}]);
```

Comment:
"Adding a model should trigger a sort". Seemed better to _only_ add a model, as the mixed order case is handled in the [test above](https://github.com/jashkenas/backbone/pull/3663/files#diff-e95d8d577a70e10552b2cdfe9a7b1178R1549).